processing/src/main/java/org/apache/druid/query/aggregation/AggregatorUtil.java

```diff
@@ -99,6 +99,9 @@
   public static final byte STRING_FIRST_CACHE_TYPE_ID = 0x2B;
   public static final byte STRING_LAST_CACHE_TYPE_ID = 0x2C;
 
+  public static final byte BLOOM_FILTER_CACHE_TYPE_ID = 0x30;
```

Comment:
0x30 and 0x31 are already in use. Also position those variables in the end of the list.

Reply:
Oops, originally did this work in an older branch where these didn't exist yet I think, will fix.

Reply:
Could you make this kind of mistake impossible in the future by moving all codes in a enum with `byte id;` field, and adding a static initializer that checks that all enum constants have different codes?

Also `CacheKeyBuilder`'s constructor to accept the enum constant instead of `byte` param to prohibit bypassing this enum